scripts/setup

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+# @license
+# This file is part of Bott.
+#
+# This project is dual-licensed:
+# - Non-commercial use: AGPLv3 (see LICENSE file for full text).
+# - Commercial use: Proprietary License (contact D@nielLaCos.se for details).
+#
+# Copyright (C) 2025 DanielLaCos.se
+
+set -euo pipefail
+
+# Get script directory and source utilities
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+source "$SCRIPT_DIR/utils/log.sh"
+source "$SCRIPT_DIR/utils/gcloud.sh"
+
+# Main setup function
+main() {
+  log_info "Bott Setup Script"
+  log_info "================="
+  
+  # Check if running on macOS
+  if [[ "$OSTYPE" == "darwin"* ]]; then
+    log_info "Detected macOS"
+    
+    # Check if brew is installed
+    if ! command -v brew &> /dev/null; then
+      log_error "Homebrew is not installed. Please install it from https://brew.sh"
```

Comment:
you can use the brew setup script /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"

but maybe confirm with the user first